vaadin-combo-box-flow/pom.xml

```diff
@@ -46,13 +46,13 @@
         <dependency>
             <groupId>org.webjars.bowergithub.vaadin</groupId>
             <artifactId>vaadin-combo-box</artifactId>
-            <version>4.2.0</version>
+            <version>4.2.2</version>
```

Comment:
please include 4.2.4 released today, as it includes 2 fixes for Flow counterpart (for both V12 and V13)

Reply:
When are you going to update `vaadin-combo-box` version to `4.2.4` in the `platform`? Based on the current version in the platform we haven't used the latest version (`4.2.4`).

Reply:
I see. Will submit a PR to the platform ASAP.

Reply:
Great. Thank you. We will update the version to 4.2.4 then.

Reply:
Ah, I thought there were some problems with it. I must have misunderstood.